src/protagonist/Portal/Pages/Index.cshtml

```diff
@@ -1,205 +1,124 @@
+                    <div>
+                        <h4 class="fw-bold mb-0">
+                            <a class="text-decoration-none" asp-page="/Queue/Index">Queue</a>
+                        </h4>
+                        <p>Paragraph of text beneath the heading to explain the heading.</p>
+                    </div>
+                </div>
+                -->
+                <div class="col d-flex align-items-start">
+                    <i class="text-muted flex-shrink-0 me-3" data-feather="folder"></i>
+                    <div>
+                        <h4 class="fw-bold mb-0">
+                            <a class="text-decoration-none" asp-page="/Spaces/Index">Spaces</a>
+                        </h4>
+                        <p>View and manage spaces</p>
+                    </div>
+                </div>
+                <!--
+                <div class="col d-flex align-items-start">
+                    <i class="text-muted flex-shrink-0 me-3" data-feather="hard-drive"></i>
+                    <div>
+                        <h4 class="fw-bold mb-0">
+                            <a class="text-decoration-none" asp-page="/Storage/Index">Storage</a>
+                        </h4>
+                        <p>Paragraph of text beneath the heading to explain the heading.</p>
+                    </div>
+                </div>
+                <div class="col d-flex align-items-start">
+                    <i class="text-muted flex-shrink-0 me-3" data-feather="globe"></i>
+                    <div>
+                        <h4 class="fw-bold mb-0">
+                            <a class="text-decoration-none" asp-page="/Origins/Index">Origins</a>
+                        </h4>
+                        <p>Paragraph of text beneath the heading to explain the heading.</p>
+                    </div>
+                </div>
+                
+           
+                <div class="col d-flex align-items-start">
+                    <i class="text-muted flex-shrink-0 me-3" data-feather="lock"></i>
+                    <div>
+                        <h4 class="fw-bold mb-0">
+                            <a class="text-decoration-none" asp-page="/Auth/Index">Access Control</a>
+                        </h4>
+                        <p>Paragraph of text beneath the heading to explain the heading.</p>
+                    </div>
+                </div>
+               
+                <div class="col d-flex align-items-start">
+                    <i class="text-muted flex-shrink-0 me-3" data-feather="search"></i>
+                    <div>
+                        <h4 class="fw-bold mb-0">
+                            <a class="text-decoration-none" asp-page="/Queries/Index">Named queries</a>
+                        </h4>
+                        <p>Paragraph of text beneath the heading to explain the heading.</p>
+                    </div>
+                </div>
+                -->

```

Comment:
commented out code

Reply:
Removed in [fc69e75](https://github.com/dlcs/protagonist/pull/640/commits/fc69e7553fd562d307aa1a6bc4b797d9a89a6b42)